src/external/VTK.jl

```diff
@@ -27,10 +27,11 @@ function save_scfres(filename::AbstractString, scfres::NamedTuple, format::Val{S
     vtkfile = vtk_grid(filename, grid)
     
     # Storing the Bloch Waves in Real space
-    for i in 1:length(basis.kpoints)
-        for j in 1:size(scfres.ψ[1])[2]
-            vtkfile["ψ_real_k$(i)_band$(j)"] = real.(G_to_r(basis, basis.kpoints[i], scfres.ψ[i][:,j]))
-            vtkfile["ψ_imag_k$(i)_band$(j)"] = imag.(G_to_r(basis, basis.kpoints[i], scfres.ψ[i][:,j]))
+    for ik in 1:length(basis.kpoints)
+        for iband in 1:size(scfres.ψ[1])[2]
+            ψ = G_to_r(basis, basis.kpoints[ik], scfres.ψ[ik][:,iband])
```

Comment:
That variable name is a little misleading (it's not the full wave function). Better call it `ψnk_real`

Reply:
I kept the name  `ψ_real` to keep the naming consistent with the next few lines.

Reply:
Actually ... it's more the other way round. The symbols stored in the VTK file are a bit confusing. The reason is that `ψ` refers to all orbitals in DFTK and we use `ψk` for the orbitals at a particular `k`-Point (so `ψ[ik]`) and `ψnk` for `ψ[ik][:, iband]`.